springfox-swagger-ui/src/web/.babelrc

```diff
@@ -4,6 +4,7 @@
       "targets": {
         "browsers": ["last 2 versions"]
       }
-    }]
+    }],
+    "stage-3"
```

Comment:
Why do we need this?

Reply:
@dilipkrish I think they wants to use [object-rest-spread](https://github.com/springfox/springfox/pull/2434/files#diff-9c74f68d3c9c52c2ea4647ddfb8a3183R14). 
```bash
...request.headers,
```
- [Babel > stage-3 > transform-object-rest-spread](https://babeljs.io/docs/en/babel-preset-stage-3.html)

Reply:
As @kasecato said.

I will push a new commit to refactor it. If you prefer this way, I will revert the commit.